views/item_list.pug

```diff
@@ -13,4 +13,23 @@ block content
     else
       li There are no #{type}s to display.
 
-  a(href='/edit/' + type + '/create') Create #{type}
\ No newline at end of file
+  a(href='/edit/' + type + '/create') Create #{type}
+  <br>
+
+  //- if type === 'preset'
+  //-   form(action="#", method='GET')
+  //-   button(type='submit' class="btn.btn-default") Download presets
```

Comment:
These are the download button code commented out. I kept it because you initially told me to make buttons but I didn't find them visually pleasing so I kept the button  code just in case you wanted it to be button. I'll delete it now that we agreed that having links are fine.